tests/Nox.Types.Tests/EntityFrameworkTests/NoxTypesEntityFrameworkTests.cs

```diff
@@ -52,16 +52,17 @@ public void Countries_CanRead_LatLong()
             Uri = Uri.From(Sample_Uri),
             IsLandLocked = Boolean.From(false),
             DateTimeDuration = DateTimeDuration.From(days: 10, 5, 2, 1),
+            Yaml = Yaml.From("foo: bar"),
```

Comment:
Can we provide a more meaningful yaml sample, like we did with Json here https://github.com/NoxOrg/Nox.Generator/blob/main/tests/Nox.Types.Tests/EntityFrameworkTests/NoxTypesEntityFrameworkTests.cs?